### PR TITLE
Feat/blox 185 menu component

### PR DESCRIPTION
**Pull request by Kristina2103**
# Pull Request Template

Feel free to remove any section (except the checklist) from this template.
But please try to fill in as much of this as possible.

## Description

Please include a summary of the proposed changes, including relevant motivation and context.

## Type of change

Please delete options that are not relevant.

- [ ] Breaking change (fix or feature that would cause existing functionality not to work as expected)
- [ ] This change requires a documentation update
- [ ] This change requires a configuration update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.
Provide easily reproducible instructions.
List any relevant details for your test configuration.

## Checklist:

- [ ] I have performed a self-review of this code and spelling
- [ ] I have added comments in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

## Additional information

Include any additional information about these changes here.

For example, if you are making a visual change, please include BEFORE (if any) and AFTER screenshots.
A visual change could be just reordering some components on a page.
Any visual changes in the UI MUST provide these screenshots.

### CODE REVIEW

File: packages/core/src/utilities/ui.ts [hunk 1]
```diff
@@ -100,10 +101,14 @@ export const relativePosition = (
     } else {
       top = targetHeight
       left = target.offsetWidth - elementDimensions.width
+
+      if (leftMenu) {
```
Comment (aleksaToljic) [inline packages/core/src/utilities/ui.ts:105]: why was this needed?

Comment (Kristina2103) [reply packages/core/src/utilities/ui.ts:105]: It was aligning only for the right side, not working as expected

**Review by aleksaToljic — APPROVED** [review]

File: packages/core/src/utilities/ui.ts (hunk 1 above)
Comment (aleksaToljic) [reply packages/core/src/utilities/ui.ts:105]: hm, alright let's keep this change in mind, in the future, this methods are really hard to tweak.
